pytorch_lightning/metrics/classification.py

```diff
@@ -87,6 +88,61 @@ def forward(self, pred: torch.Tensor, target: torch.Tensor) -> torch.Tensor:
                         num_classes=self.num_classes, class_reduction=self.class_reduction)
 
 
+class AccuracyTopk(TensorMetric):
+    """
+    Computes the accuracy classification score
+
+    Example:
+
+        >>> pred = torch.tensor([0, 1, 2, 3])
+        >>> target = torch.tensor([0, 1, 2, 2])
+        >>> metric = AccuracyTopk(topk=1)
+        >>> metric(pred, target).item()
+        0.75
+
+    """
+
+    def __init__(
+        self,
+        num_classes: Optional[int] = None,
+        class_reduction: str = 'micro',
+        topk: int = 1,
```

Comment:
since this is a top accuracy, this should not have a default parameter but instead be the first parameter to give (without a default). This may change if we integrate with other accuracy though